extras/cloudsql/kubernetes-manifests/balance-reader.yaml

```diff
@@ -92,7 +92,7 @@ spec:
           limits:
             cpu: "200m"
```

Comment:
These CPU/Memory limits are pretty low as well - have you verified these that this is enough for the proxy and this app?

Reply:
Those are just the limits of the proxy side-car; the app has different limits set further above.